lang/python/setup.py

```diff
@@ -50,10 +50,14 @@
 
 setup(name='wiredtiger', version=wt_ver,
     ext_modules=[Extension('_wiredtiger',
-                [os.path.join(dir, 'wiredtiger_wrap.c')],
+                [os.path.join(dir, 'cmake_pip_build', 'lang', 'python', 'CMakeFiles', '__wiredtiger.dir', 'wiredtigerPYTHON_wrap.c')],
```

Comment:
@ddanderson I don't know how to fix this. I just realized that auto tools actually depends on this:

```
_wiredtiger.so: $(top_builddir)/libwiredtiger.la $(PYSRC)/wiredtiger_wrap.c
	(cd $(PYSRC) && \
	    $(PYTHON) setup.py build_ext -f -b $(abs_builddir) $(PYDIRS))
```

Reply:
@alisonfel I cannot update the path in setup.py. Not sure what I can do here.

Reply:
@quchenhao  What type of error occurs when you try to include the path in `setup.py`?